api/src/pcapi/core/finance/api.py

```diff
@@ -651,3 +657,155 @@ def edit_business_unit(business_unit: models.BusinessUnit, siret: str) -> None:
     business_unit.siret = siret
     db.session.add(business_unit)
     db.session.commit()
+
+
+def _find_reimbursement_rule(rule_reference: [str, int]) -> payments_models.ReimbursementRule:
+    # regular rule description
+    if isinstance(rule_reference, str):
+        for regular_rule in reimbursement.REGULAR_RULES:
+            if rule_reference == regular_rule.description:
+                return regular_rule
+    # CustomReimbursementRule.id
+    return payments_models.CustomReimbursementRule.query.get(rule_reference)
+
+
+def _make_invoice_line(invoice: models.Invoice, group: conf.RuleGroups, pricings: list):
+    reimbursed_amount = 0
+    pricings_lines = []
+    for line in (pricing.lines for pricing in pricings):
+        pricings_lines.append(line)
+    flat_lines = list(itertools.chain.from_iterable(pricings_lines))
```

Comment:
(détail) un truc m'échappe, ne peux-tu pas écrire directement : 

```suggestion
    flat_lines = list(itertools.chain.from_iterable(pricing.lines for pricing in pricings))
```